src/i965_device_info.c

```diff
@@ -309,6 +317,8 @@ static struct hw_codec_info chv_hw_codec_info = {
     .has_h264_mvc_encoding = 1,
     .has_hevc_decoding = 1,
 
+    .h264_brc_mode = VA_RC_CQP | VA_RC_CBR,
```

Comment:
@pengche1 a simple VBR was added on prev-skl by the commit below:

commit f804a502699c05b6a61afac833fd465df16b32c8
Author: Mark Thompson <sw@jkqxz.net>
Date:   Tue Jan 10 00:04:43 2017 +0000

H.264 encoder: add a simple reactive VBR rate control mode